plugins/processors/noise/noise.go

```diff
@@ -0,0 +1,157 @@
+package noise
+
+import (
+	"fmt"
+	"math"
+	"reflect"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/plugins/processors"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+const (
+	defaultScale     = 1.0
+	defaultMin       = -1.0
+	defaultMax       = 1.0
+	defaultMu        = 0.0
+	defaultNoiseType = "laplacian"
+)
+
+const sampleConfig = `
+  [[processors.noise]]
```

Comment:
The section name `[[processors.noise]]` should not be contained in `sampleConfig`.